pkg/commands/exec_live_win.go

```diff
@@ -0,0 +1,9 @@
+// +build windows
+
+package commands
+
+// RunCommandWithOutputLiveWrapper runs a command live but because of windows compatibility this command can't be ran there
+// TODO: Remove this hack and replace it with a propper way to run commands live on windows
+func RunCommandWithOutputLiveWrapper(c *OSCommand, command string, output func(string) string) (errorMessage string, codeError error) {
+	return errorMessage, c.RunCommand(command)
```

Comment:
Yes this just runs the "old" way without interaction.  
Thx for reminding btw i see a problem here because errorMessage is a empty string that will be shown when there is an error this way the user will see an empty error on the screen